### ISSUE TITLE
[pni] homepage: don't skip heading levels

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4643]
Headings should not skip levels, so this h3 should be an h2, or not use heading markup:

![](https://user-images.githubusercontent.com/177243/81863675-71dca880-9520-11ea-90bd-f2e991fde5b2.png)
